react/features/prejoin/components/DropdownButton.js

```diff
@@ -59,7 +59,7 @@ const styles = theme => {
             display: 'flex',
             height: 40,
             fontSize: 15,
-            lineHeight: 24,
+            lineHeight: '24px',
```

Comment:
why do we use px here but not for the height for example?

Reply:
mUI quirk, for lineHeight it takes em as default. This was causing a pretty ugly bug on the dropdown as the line height was over 300px for the buttons labels.

Reply:
All I hear is some CSS gibberish, so I trust you :-)